crates/cxx-qt-lib/src/core/qt.rs

```diff
@@ -22,6 +22,22 @@ mod ffi {
         CaseSensitive,
     }
 
+    #[repr(i32)]
+    enum ConnectionType {
```

Comment:
Some other ideas:
1. Use `on_MY_SIGNAL` and `on_MY_SIGNAL_with_type` - because we don't have control over the signal name, this might be easy to overlook though.
2. Use `on_MY_SIGNAL` for Auto-type connections, and `connect_MY_SIGNAL` for the "advanced" API with options.

Reply:
Number 2) here i think is best, i'll try that..

Reply:
Done

Reply:
That cleaned up nicely